pkg/analyser/analyzer.go

```diff
@@ -72,16 +75,18 @@ func (a Analyzer) Analyze(remoteResources, resourcesFromState []*resource.Resour
 		}
 
 		if !found {
-			analysis.AddDeleted(stateRes)
+			if analysis.Options().OnlyManaged || !hasOneOnlyFlag {
```

Comment:
`if !analysis.Options().OnlyUnmanaged {` should be enough here nope ? 

I don't think we need `hasOneOnlyFlag` do we ?

Reply:
Absolutely right, good catch !